CMakeLists.txt

```diff
@@ -68,7 +69,11 @@ OPTION(BUILD_TESTS "xtensor-zarr test suite" OFF)
 OPTION(DOWNLOAD_GTEST "build gtest from downloaded sources" OFF)
 OPTION(DOWNLOAD_GBENCHMARK "download google benchmark and build from source" ON)
 
-add_library(xtensor-zarr INTERFACE)
+add_library(xtensor-zarr SHARED
+    ${XTENSOR_ZARR_SRC_DIR}/shared.cpp
+)
+
+add_library(libxtensor-zarr INTERFACE)
```

Comment:
It should be the opposite, `libxtensor-zarr` for the shared object and `xtensor-zarr` for the pure interface.